src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/HomeController.java

```diff
@@ -43,10 +43,12 @@ public static String getHomePageObjectJSON(String baseUrl) throws JsonProcessing
         resultMap.put("greeting","Greetings from Spring Boot!");
```

Comment:
Thank you for your work, I appreciate it. I am wondering maybe if you can submit separate pull requests for different issues. Like here you  are addressing issue 14, but you also modified file for issue 3. Maybe we can all work on issue 3 together when everyone is done with their assigned work.

Reply:
Also maybe you can try to look through your code again to see if you can pass those automated pass before we proceed furthur with code review and merging this pull request

Reply:
I updated the code so that I do not modify the file for issue 3. The automated checks now pass.